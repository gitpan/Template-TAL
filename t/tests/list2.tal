<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    <div tal:repeat="colour1 path:colours">
      <div tal:repeat="colour2 path:colours">
        #<span tal:replace="repeat/colour1/index" />
        .<span tal:replace="repeat/colour2/index" />
        <span tal:replace="colour1">Colour of the rainbow</span>
        <span tal:replace="colour2">Colour of the rainbow</span>
      </div>
    </div>
    <div tal:repeat="colour1 path:colours" tal:omit-tag="string:1">
      <span tal:content="colour1" />
    </div>
  </body>
</html>
