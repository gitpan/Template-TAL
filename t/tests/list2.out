<html>
  <body>
    <div>
      <div>#0 .0 red red</div>
      <div>#0 .1 red green</div>
      <div>#0 .2 red blue</div>
    </div>
    <div>
      <div>#1 .0 green red</div>
      <div>#1 .1 green green</div>
      <div>#1 .2 green blue</div>
    </div>
    <div>
      <div>#2 .0 blue red</div>
      <div>#2 .1 blue green</div>
      <div>#2 .2 blue blue</div>
    </div>

    <span>red</span>
    <span>green</span>
    <span>blue</span>

  </body>
</html>
